packages/size-limit/run.js

```diff
@@ -43,10 +50,20 @@ module.exports = async process => {
 
     config = await getConfig(plugins, process, args, pkg)
 
-    await calc(plugins, config, ora)
+    if (hasArg('--watch')) {
+      let watcher = chokidar.watch([
+        '**/*.js',
+        'package.json'
+      ], {
+        ignored: '**/node_modules/**'
+      })
+      watcher.on('change', async () => {
```

Comment:
If the user presses `Ctrl+S` twice very fast, chokidar will start 2 Size Limit’s `mainCalc` in parallel.

We need a way to throttle it. We can look at a solution in chokidar or copy my solution from [print-snapshots](https://github.com/ai/print-snapshots/blob/master/watch.js#L40).

Reply:
I've fixed this. The only one question. Why do you set delay to 200ms? Do you need really wait so long or it's enough to wait next cycle like ( setTimeout(() => ..., 0) )?

Reply:
You are right. We need to wait for full circle. `print-snapshots` wait for full circle and 200ms.

https://github.com/ai/print-snapshots/blob/master/watch.js#L14-L16